src/components/checkbox/checkbox.scss

```diff
@@ -250,6 +250,10 @@ $_md-checkbox-indeterminate-checked-easing-function: cubic-bezier(0.14, 0, 0, 1)
   }
 }
 
+.md-checkbox-layout label {
+  line-height: 24px;
```

Comment:
I'm not sure if we should be setting line-height. I would think that the checkbox should just use whatever line height is already applied. 

@hansl, I think you will have thoughts on this